src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -411,11 +413,11 @@ void preferences_dialog::post_build(window& window)
 		font_scaling, set_font_scaling);
 
 	/* SELECT THEME */
-	connect_signal_mouse_left_click(
-			find_widget<button>(&window, "choose_theme", false),
-			bind_void(&show_theme_dialog,
-			std::ref(window.video())));
-
+	button *theme_button = find_widget<button>(&window, "choose_theme", false, false);
+	if (theme_button != nullptr) {
+		connect_signal_mouse_left_click(
+				*theme_button, bind_void(&show_theme_dialog, std::ref(window.video())));
+	}
```

Comment:
We'll definitely get here after resolving more pressing issues, like GUI with 150% font size, and general mobile-friendly GUI :)
As for now, I follow the principle declared by Joel: https://www.joelonsoftware.com/2006/11/21/choices-headaches/ - "whatever we can live without, we should".